### ISSUE TITLE
Request for inclusion for sklearn-evaluation

### ISSUE THREAD

**Issue by edublancas** [scikit-learn-contrib/scikit-learn-contrib#13]
Request for project inclusion in scikit-learn-contrib

* Project name: sklearn-evaluation
* Project description: Model evaluation (mostly plots for now) and report generation
* Authors: Eduardo Blancas Reyes
* Current repository: https://github.com/edublancas/sklearn-evaluation
* Requirements:
 * [x] scikit-learn compatible (check_estimator passed) - does not apply
 * [x] Documentation (guide, API reference, example gallery)
 * [x] Unit tests (coverage: 94%)
 * [x] Python3 compatible
 * [x] PEP8 compliant
 * [x] Continuous integration

**Comment by amueller** [comment]
Can you maybe say a bit more about the content? I have PR for a confusion matrix plot to sklearn.
That also allows some easy visualization of grid-search results, which I think is important.

I would actually like to have some more ready-made functions for PR curves and ROC curves and feature importances.
Getting them "upstream" will take longer and a bit of discussion, but I think it will be useful.

I'm not sure what the best route is. I'm pretty busy right now but I'd love to see this in the main repo.
We can include here and see what we can merge upstream maybe?

**Comment by edublancas** [comment]
Sure, I updated the description.

**Comment by wdm0006** [comment]
+1 from me for inclusion, looks to be a very useful library, in particularly the report generation portion.

**Comment by jnothman** [comment]
This is a bit different from what tends to be in scikit-learn-contrib. It is more comparable to something like eli5. Do we want this kind of thing in contrib, or put them on the related projects page?

**Comment by glemaitre** [comment]
> This is a bit different from what tends to be in scikit-learn-contrib.

I tend to think that it could be useful since that it could provide some plotting functionality to scikit-learn. I also think that having a tool similar to https://github.com/pandas-profiling/pandas-profiling could be good for the scikit-learn-contrib.
